### PR TITLE
Accessibility add link name

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/kubesphere/website/issues/2392

### CODE REVIEW

**Comment by Arhell** [comment]
cc @FeynmanZhou

**Comment by FeynmanZhou** [comment]
Thanks for your contribution! @Arhell 
Hi @weili520 @harrisonliu5 , I see @Arhell has fixed multiple 
accessibility issues and added aria-label for some links. I am not familiar with the front-end. I am curious if we need to do a whole check to the KubeSphere website for this kind of issue?

File: layouts/partials/downloadButton.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<a href="{{ "docs/quick-start/minimal-kubesphere-on-k8s/" | relLangURL }}" target="_blank" rel="noopener noreferrer">
+<a href="{{ "docs/quick-start/minimal-kubesphere-on-k8s/" | relLangURL }}" aria-label="Install on Kubernetes" target="_blank" rel="noopener noreferrer">
   <button class='download-btn download-btn1 common-btn'>{{ i18n "Install on Kubernetes" }}</button>
 </a>
-<a href="{{ "docs/quick-start/all-in-one-on-linux/" | relLangURL }}" target="_blank" rel="noopener noreferrer">
+<a href="{{ "docs/quick-start/all-in-one-on-linux/" | relLangURL }}" aria-label="Install on Linux" target="_blank" rel="noopener noreferrer">
```
Comment (weili520) [inline layouts/partials/downloadButton.html:4]: I think we‘d better use i18n to transform it.

Comment (FeynmanZhou) [reply layouts/partials/downloadButton.html:4]: @Arhell PTAL, thanks.

Comment (Arhell) [reply layouts/partials/downloadButton.html]: done

**Pull request merged**